### DISCUSSION TITLE
Turn off file hashing.

### DISCUSSION THREAD

**Discussion by Nigel2392** [wailsapp/wails#2213] [Q&A]
Is there a way to turn off the file name hashing that occurs when building the application?
```
/frontend/dist/
index.9fc7c794.js -> index.js
main.f80c4559.wasm -> main.wasm
```
I cannot find any documentation on this particular issue.

**Comment by brys0** [comment]
This isn't controlled by Wails. This is probably controlled by your frontend tooling, most likely vite/rollup. For your vite config, in rollup output, you can ask for a file template name that doesn't involve the hash part of it.

You can see that here: https://stackoverflow.com/questions/69614671/vite-without-hash-in-filename
